simulation/README.md

```diff
@@ -8,13 +8,11 @@ Additionally, it publishes a [TwistStamped](http://docs.ros.org/melodic/api/geom
 This readme assumes that you are able to mount the git repo in the docker container, and that you have the most recent Docker image pulled. 
 
 ### Setting up the Docker Container
-You will need a Docker container with port 8080 forwarded. Additionally, you will need to use the simulation tag for the Docker container. Run the command that you normally do to create your Docker container according to the [documentation repo](https://github.com/DukeRobotics/documentation/tree/master/docker), but with `-p 8080:8080` added, and with `:simulation` at the end.
+You will need a Docker container with port 8080 forwarded. You will be using the landside container. Run the command that you normally do to create your Docker container, but with `-p 8080:8080` added.
     
 For example, you might run the command:
 
-`docker run -td -p 2200:2200 -p 8080:8080 --mount type=bind,source=C:\Users\Eric\Documents\Robotics\CS,target=/home/duke/dev/robosub-ros/src dukerobotics/robosub-ros:simulation`
-    
-[Here's](https://github.com/DukeRobotics/documentation/tree/master/docker) the link to the documentation repo with the Docker commands, if you need to look yours up.
+`docker run -td -p 2201:2201 -p 8080:8080 --mount type=bind,source=[source path],target=[mount path] dukerobotics/robosub-ros:landside`
 
```

Comment:
You can delete all of this, the docker compose file takes care of it. If you want to say something, you can say start up the landside container as usual, or as located in the main readme.

Reply:
Fixed.